VPATH installs do not find copyright file

Out-of-source (VPATH) installation fails with:
```
-- Up-to-date: /usr/include/libks/cmake/AddLibBacktrace.cmake
CMake Error at cmake_install.cmake:114 (file):
  file INSTALL cannot find "/git/freeswitch/libks/build/copyright": No such
  file or directory.
```
The file is not in the build directory, but in the source directory.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -55,7 +55,7 @@ jobs:
 
   deb-mirror:
     name: "DEB-MIRROR"
-    if: ${{ github.event_name != 'pull_request' }}
+    if: ${{ github.event_name != 'pull_request' && github.repository == 'signalwire/libks' }}
     needs:
       - deb
     runs-on: ubuntu-latest
@@ -104,7 +104,7 @@ jobs:
 
   meta:
     name: "Publish build data to meta-repo"
-    if: ${{ github.event_name != 'pull_request' }}
+    if: ${{ github.event_name != 'pull_request' && github.repository == 'signalwire/libks' }}
     needs:
       - deb
       - deb-mirror

